internal/tasks/resourcesync.go

```diff
@@ -240,6 +245,25 @@ func (r *ResourceSync) parseAndValidateResources(rs *api.ResourceSync, repo *api
 		return nil, err
 
 	}
+	//ResourceUpdateFound (include the resource name, the operation - update/delete, and the git commit in the message and details)
+	updateDetails := &api.ResourceUpdatedDetails{
+		UpdatedFields: []api.ResourceUpdatedDetailsUpdatedFields{
+			api.ResourceUpdatedDetailsUpdatedFields(fmt.Sprintf("resource %s", *rs.Metadata.Name)),
+			api.ResourceUpdatedDetailsUpdatedFields(fmt.Sprintf("op %s", "???")), // TODO: where to get this?
+			api.ResourceUpdatedDetailsUpdatedFields(fmt.Sprintf("commit %s", hash)),
+		},
+	}
+	if r.serviceHandler != nil {
+		r.serviceHandler.CreateEvent(ctx,
+			r.serviceHandler.CreateGenericEvent(ctx, api.RepositoryKind, resourceSyncName, resourceSyncTask, api.StatusOK(),
```

Comment:
There are similar issues for ResourceSync that for Repository events:

1. involvedObject.name is "resourcesync" and not the actual RS name.
2. metadata.name is also invalid according to the k8s spec.
3. Needs solving the "op ???" TODO as it appears in updatedFields and message.